test/unit/alignment/matrix/detail/two_dimensional_matrix_test.cpp

```diff
@@ -145,6 +145,42 @@ TYPED_TEST(two_dimensional_matrix_test, at)
     EXPECT_THROW((matrix.at({row_index_type{2u}, column_index_type{4u}})), std::invalid_argument);
 }
 
+TYPED_TEST(two_dimensional_matrix_test, conversion)
+{
+    using matrix_type = typename TestFixture::matrix_type;
+
+    matrix_type matrix{number_rows{3}, number_cols{4}};
+    auto it = std::copy(this->expected_range.begin(), this->expected_range.end(), matrix.begin());
```

Comment:
```suggestion
    std::ranges::copy(this->expected_range, matrix.begin());
```
This would apply to more locations in this file.
Unless I missed something, in which case: Why `std::copy` and why store the returned iterator when you are not using it...

Reply:
Good question. Must be some leftover or copy'n'paste issue